dts/msm8974/msm8974-samsung-klte.dts

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+/include/ "skeleton.dtsi"
+
+/ {
+	// This is used by the bootloader to find the correct DTB
+
+	// qcom,board-id = <0xCE08FF01 8>;
+	// ^^ no qcom,board-id in downstream DTB; downstream DTB has only this:
+	qcom,msm-id = <0xC208FF01 14 0x10000>;
+	// model = "SAMSUNG K PROJECT REV14";
```

Comment:
I think that's fine, that's the older way to specify those. Can you remove the comments here?

Reply:
:ok: